src/FlameGraph.js

```diff
@@ -12,6 +12,8 @@ type Props = {|
   data: ChartData,
   height: number,
   onChange?: (chartNode: ChartNode, uid: any) => void,
+  onMouseOut?: (e: any, node: RawData) => void,
+  onMouseOver?: (e: any, node: RawData) => void,
```

Comment:
Unsure if we can be more specific on the event type here; it's a MouseEvent from React in TypeScript terms, but I think these are Flow types instead? I'll fix this up to proper typing provided the rest of the PR passes review.

Reply:
I think the Flow type you're looking for here is `SyntheticMouseEvent<*>`